### PR TITLE
BrightSign.BrightAuthor.5.0 version 5.0.1.2

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102148)

### CODE REVIEW

File: manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor5.0
```
Comment (Trenly) [inline manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.installer.yaml:4]:
```suggestion
PackageIdentifier: BrightSign.BrightAuthor.5
```

File: manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor5.0
```
Comment (Trenly) [inline manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: BrightSign.BrightAuthor.5
```

File: manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor5.0
```
Comment (Trenly) [inline manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.yaml:4]:
```suggestion
PackageIdentifier: BrightSign.BrightAuthor.5
```

File: manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor5.0
+PackageVersion: 5.0.1.2
+PackageLocale: en-US
+Publisher: BrightSign
+PackageName: BrightAuthor 5.0
+License: Proprietary
+Copyright: Copyright (c) BrightSign. All rights reserved.
+ShortDescription: Create programs for BrightSign appliences
```
Comment (Trenly) [inline manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Create programs for BrightSign appliances
```

File: manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor5.0
+PackageVersion: 5.0.1.2
+Installers:
+- InstallerUrl: https://brightsignbiz.s3.amazonaws.com/brightauthor/5.0/5.0.1.2/BrightAuthorSetup-5.0.1.2.exe
+  Architecture: x86
+  InstallerType: burn
+  InstallerSha256: DD425F33930DF791858A415D70ADD411F7E78421D82F8123261316E413012E39
```
Comment (Trenly) [inline manifests/b/BrightSign/BrightAuthor5/0/5.0.1.2/BrightSign.BrightAuthor5.0.installer.yaml:10]:
```suggestion
  InstallerSha256: DD425F33930DF791858A415D70ADD411F7E78421D82F8123261316E413012E39
  ProductCode: '{bfdcc667-fe53-4ed1-825f-4b6435f0fa7c}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by upintheairsheep** [comment]
I wish there was an easier way to move folders in GitHub

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\BrightSign\BrightAuthor5\0\5.0.1.2', expected 'manifests\b\BrightSign\BrightAuthor\5\5.0.1.2' 

The file path has an extra `\0`.

**Comment by upintheairsheep** [comment]
> Automatic Validation failed with:
> 
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\BrightSign\BrightAuthor5\0\5.0.1.2', expected 'manifests\b\BrightSign\BrightAuthor\5\5.0.1.2'
> 
> The file path has an extra `\0`.

How can I remove it?

**Comment by stephengillie** [comment]
> How can I remove it?

- Switch to this PR's branch (upintheairsheep:autogenerated/BrightSign.BrightAuthor5.0/a32b123a-ec6c-40c2-a564-ad97a89c63e3) in your Git client by checking it out. Then rename the folder in your OS, and commit the changes. It should automatically show up here.

File: manifests/b/BrightSign/BrightAuthor/5/0/5.0.1.2/BrightSign.BrightAuthor.5.0.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor.5
```
Comment (stephengillie) [inline manifests/b/BrightSign/BrightAuthor/5/0/5.0.1.2/BrightSign.BrightAuthor.5.0.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: BrightSign.BrightAuthor.5.0
```

File: manifests/b/BrightSign/BrightAuthor/5/0/5.0.1.2/BrightSign.BrightAuthor.5.0.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BrightAuthor.5
```
Comment (stephengillie) [inline manifests/b/BrightSign/BrightAuthor/5/0/5.0.1.2/BrightSign.BrightAuthor.5.0.yaml:4]:
```suggestion
PackageIdentifier: BrightSign.BrightAuthor.5.0
```

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
> 'manifests\b\BrightSign\BrightAuthor\5.0\5.0.1.2', expected 
> 'manifests\b\BrightSign\BrightAuthor\5\0\5.0.1.2'

**Review by stephengillie — APPROVED** [review]

**Pull request merged**